Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,16 +1,363 @@
+            {
+                paddingLine += " ";
+            }
+            paddingLine += "|";
+
+            for (var i = 0; i < padding; i++)
+            {
+                frame[i + 1] = paddingLine;
+                frame[frame.Length - 2 - i] = paddingLine;
+            }
+
+
+            //boddy
+            for (var i = 0; i < arr.Length; i++)
+            {
+                //begining of each text line
+                frame[i + 1 + padding] = $"{Environment.NewLine}|";
+                //add padding
+                for (int j = 0; j < padding; j++)
+                {
+                    frame[i + 1 + padding] += " ";
+                }
+                //add text
+                frame[i + 1 + padding] += arr[i];
+
+                //ending and padding
+                for (var k = 0; k < longerLineSize + padding - arr[i].Length; k++)
+                {
+                    frame[i + 1 + padding] += " ";
+                }
+                frame[i + 1 + padding] += "|";
+            }
+
+            return String.Join("", frame);
+        }
+
+        static string ComposeListString(string[] items)
+        {
+            if (items == null)
+            {
+                return "";
+            }
+            int length = items.Length;
+            switch (length)
+            {
+                case 0:
+                    return "";
+                case 1:
+                    return items[0];
+                case 2:
+                    return string.Join(" and ", items);
+                default:
+                    var firstsArr = new string[items.Length - 1];
+                    for (int i = 0; i < firstsArr.Length; i++)
+                    {
+                        firstsArr[i] = items[i];
+                    }
+
+                    string firsts = string.Join(", ", firstsArr);
+                    var last = $" and {items[^1]}";
+                    return firsts + last;
+            }
+        }
+
+        //static string ComposeListString(string[] items) //this is an optimization of my method that Bing chat sugested 
+        //{
+        //    if (items == null || items.Length == 0)
+        //    {
+        //        return "";
+        //    }
+        //    var sb = new StringBuilder();
+        //    sb.AppendJoin(", ", items[..^1]); // append all items except the last one
+        //    if (items.Length > 1)
+        //    {
+        //        sb.Append(" and "); // append the conjunction
+        //    }
+        //    sb.Append(items[items.Length - 1]); // append the last item
+        //    return sb.ToString();
+        //}
+
+        static string[] AppendToStringArray(string[] items, string item)
+        {
+            string[] newArray = new string[items.Length + 1];
+            for (int i = 0; i < items.Length; i++)
+            {
+                newArray[i] = items[i];
+            }
+            newArray[^1] = item;
+            return newArray;
+
+        }
+
     }
 }
```

Comment:
Should this belong in Program.cs? Or should it be moved to its own components? 360 lines - is that many lines of code for one file?